sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/OneCollectorChannelListener.java

```diff
@@ -80,7 +90,8 @@ public void onGroupAdded(@NonNull String groupName) {
             return;
         }
         String oneCollectorGroupName = getOneCollectorGroupName(groupName);
-        mChannel.addGroup(oneCollectorGroupName, ONE_COLLECTOR_TRIGGER_COUNT, ONE_COLLECTOR_TRIGGER_INTERVAL, ONE_COLLECTOR_TRIGGER_MAX_PARALLEL_REQUESTS, null, null);
+        Ingestion ingestion = new OneCollectorIngestion(mContext, mLogSerializer);
```

Comment:
Move ingestion to constructor with a field named mIngestion so that we share same instance for multiple groups, forgot about that. In that case we won't need to keep mContext as a field.